examples/angular/README.md

```diff
@@ -0,0 +1,29 @@
+# Angular Example
+
+This is a demo to explain the use of ArmoniK Api for Angular.
+
+Please, refer to the documentation to get more information about the usage of the library. We write a special guide to help you to use ArmoniK Api with Angular.
```

Comment:
Please, refer to the documentation to get more information about the usage of the library. We wrote a guide on how to use ArmoniK Api with Angular.